src/main/java/bizbook/ui/CommandHistory.java

```diff
@@ -47,8 +48,12 @@ public String getPreviousCommand() {
      * @return the next command, or an empty string if at the end of the history
      */
     public String getNextCommand() {
-        if (currentIndex >= 0 && currentIndex < history.size()) {
-            return history.get(currentIndex++);
+        assert currentIndex >= -1;
```

Comment:
Maybe can add `-1` as a magic literal, since it used in the initialization and here?